code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -24,7 +24,27 @@
 	var/isbroken = 0
 	light_range = 5
 	light_color = LIGHT_COLOR_RED
-	var/obj/item/wepon = null
+	var/last_check
+
+/obj/structure/cult/pylon/New()
+	..()
+	processing_objects.Add(src)
+
+/obj/structure/cult/pylon/Destroy()
+	..()
+	processing_objects.Remove(src)
+
+/obj/structure/cult/pylon/process()
+	if(!isbroken && world.time > last_check + 3 SECONDS)
+		var/list/can_see = view(src, 3)
+		last_check = world.time
+		for(var/mob/living/simple_animal/construct/C in can_see)
```

Comment:
Ah that's true, it isn't in the *```range()``` family, no.
Unless of course it's the opposite, and it's that ```block()``` is *more* optimised (since ```trange``` (we just have ```RANGE_TURFS()``` on /tg/, ```trange``` died ages ago) uses ```block()```), I wouldn't put that past byond.